releasenotes/jmri4.5.6.shtml

```diff
@@ -417,6 +417,54 @@ The <a href="https://github.com/JMRI/JMRI/issues?utf8=✓&q=is%3Aclosed&q=milest
 
    <h4>Web Access</h4>
         <ul>
+            <li></li>
```

Comment:
@petecressman it seems that you might have started from an earlier version of the release note that had already had the placeholder entries stripped out.

Could you revert the additions of these placeholder entries please?